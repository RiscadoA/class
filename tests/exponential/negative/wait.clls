proc main() {
    cut {
        println("!"); !a(b: wait);
        println("wait"); wait b; ()
        |a: ?close|
        println("?"); ?a;
        println("call 1"); call a(y);
        println("call 2"); call a(z);
        par {
            println("close 1"); close y
            ||
            println("close 2"); close z
        }
    }
};;
