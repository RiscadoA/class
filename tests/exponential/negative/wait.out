!
?
call 1
call 2
close 1
wait
close 2
wait
